pkgs/by-name/c2/c2000-cgt/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  buildFHSEnv,
+  lib,
+  stdenvNoCC,
+}:
+
+let
+  fhs = buildFHSEnv {
+    name = "fhs-shell";
+  };
+
+  package = stdenvNoCC.mkDerivation rec {
+    pname = "cgt_c2000";
+    version = "22.6.1.LTS";
+
+    filename = "ti_${pname}_${version}_linux-x64_installer.bin";
+
+    src = builtins.fetchurl {
+      url = "https://dr-download.ti.com/software-development/ide-configuration-compiler-or-debugger/MD-xqxJ05PLfM/${version}/${filename}";
```

Comment:
Is the `MD-xqxJ05PLfM` part invariant of the version?

Reply:
Yes, it's a constant across all versions.